pyccel/codegen/compiling/basic.py

```diff
@@ -238,9 +253,17 @@ def release_simple_lock(self):
 
     @property
     def accelerators(self):
-        """ Returns the names of the accelerators required to compile the file
         """
-        return self._accelerators
+        Get the names of the accelerators required to compile the file.
+
+        Return a set containing the name of all accelarators required
```

Comment:
The spellchecker does not check the code itself (this would lead to a horrible amount of errors on variable names). It could check the docstrings once they are used to generate a doc (see #1287).

Fixed 62b63831